packages/sync/src/y-webrtc/crypto.js

```diff
@@ -1,118 +1,127 @@
 // File copied as is from the y-webrtc package.
 /* eslint-disable eslint-comments/disable-enable-pair */
 /* eslint-disable eslint-comments/no-unlimited-disable */
-/* eslint-disable */ 
+/* eslint-disable */
```

Comment:
Are there any functional changes to this file or only linting? I feel like this shouldn't be a necessary change within this PR, so it'd be great if we could put this in a separate PR if it's not necessary here.